lib/THC/THCThreadLocal.h

```diff
@@ -0,0 +1,16 @@
+#ifndef THC_THREAD_LOCAL_INC
+#define THC_THREAD_LOCAL_INC
+
+#ifdef _WIN32
+typedef DWORD THCThreadLocal;
+#else
+#include <pthread.h>
```

Comment:
It might be better to add something like [this](https://github.com/torch/torch7/blob/master/lib/TH/CMakeLists.txt#L188-L193) to CMakeLists.txt if using pthread.